src/frontend/calendar/index.js

```diff
@@ -41,25 +47,33 @@ function Calendar() {
 			<div className="wporg-meeting-calendar__header">
 				<nav
 					className="wporg-meeting-calendar__btn-group"
-					aria-label={ __( 'Month navigation', 'wporg-meeting-calendar' ) }
+					aria-label={ __(
+						'Month navigation',
+						'wporg-meeting-calendar'
+					) }
 				>
 					<Button
-						isSecondary
+						variant="secondary"
```

Comment:
`isSecondary` is deprecated, `variant` is the new API, same goes for `isLink` and `isPrimary`